### ISSUE TITLE
🪣 Sign-in Reskin (For Denver)

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#6382]
## Description

This is a Bucket Ticket for the Sign-in Reskin. The first phase of our Sign-in Revamp will not entail the UX Flow updates that will require backend updates in Phases 2+ in the User Onboarding Figma. The first phase is a reskin of our current flow with an explicit mindfulness toward our new rebrand design + mobile functionality.

The entirety of the screens for Phase 1 is here: https://www.figma.com/file/I5jbkBPzNrbYMHYQJb6CNY/Onboarding?type=design&node-id=481-210&mode=dev

## Project Owner

PM: @zakhap
Designer: Jared

## Tickets

- [x] #6383
- [x] #6384
- [x] #6386 
- [x] #6387
- [ ] BLOCKED ON PRODUCT REVIEW: #6385 (NOT BLOCKING ON RELEASE, TAGGED "Low Priority" accordingly)

## Additional context

<img width="1020" alt="Screen Shot 2024-01-17 at 5 41 03 PM" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/9805e6e4-2bc9-45fb-9d55-f21f7cd8c4fc">
<img width="389" alt="email scree" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/8cc1b605-044e-4aa5-b30b-35ed8f233f58">

**Comment by CowMuon** [comment]
Blocks #6509
